### PR TITLE
Week 23 Software Updates

### PR DESCRIPTION
**Pull request by Kutkovsky**
Software updates:

- curl 7.65.0
- GNU Wget 1.20.3
- Homebrew 2.1.4
- xctool 0.3.6
- CocoaPods 1.7.0
- App Center CLI 1.1.18
- PowerShell 6.2.1
- azure-cli 2.0.64
- NVM - Installed node versions:
  v10.16.0
  v12.3.1
- GNU parallel 20190522

Android:
- Android Emulator 29.0.11

Xamarin:
- Visual Studio for Mac: 8.0.8.2

### CODE REVIEW

File: images/macos/macos-10.14-Readme.md [hunk 1]
```diff
@@ -26,7 +26,12 @@ Previously, Microsoft hosted Mac machines had JDKs pre-installed that were overl
 - Node.js 6.17.0
 - Node.js 8.11.3
```
Comment (max-lobanov) [inline images/macos/macos-10.14-Readme.md]: We don't need line `Node.js 8.11.3` because it is NVM version. Please remove.

Comment (Kutkovsky) [reply images/macos/macos-10.14-Readme.md]: @max-lobanov It installs as part of Xamarin provisionator (per their initial request)

Comment (Kutkovsky) [reply images/macos/macos-10.14-Readme.md]: Thanks, fixed

**Comment by Kutkovsky** [comment]
Will recreate another PR, sorry. Messing with conflicts, too many changes

**Pull request closed without merging**